org.gazebosim.Gazebo.yaml

```diff
@@ -0,0 +1,677 @@
+app-id: org.gazebosim.Gazebo
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mono6
```

Comment:
You need more than just this to use mono. you need to set the path and possibly copy the runtime if you need.

Crurrently, with the change below for OGRE (using cmake), the build fail because of `mcs` not being found.

Reply:
To clarify you need to set the path so that `mcs` can be found, and you need to run the install script to copy over the runtime.

This assume this is needed for the whole app because if it is just a matter to tell OGRE to ignore because it is not needed it might be easier to just do that.

(I'm try to build the whole thing locally, not just armchair reviewing)

Reply:
I think mcs is only needed for ogre, but since gazebo is not working properly at the moment, I try to build everything.

Reply:
for OGRE, I added the following config-opts: `- -DOGRE_BUILD_COMPONENT_CSHARP=FALSE` and this disabled the need for mono.

I'm pretty confident that it's not used by Gazebo as there is no trace a CIL binaries but the assembly from OGRE that no longer exist with this option, or the mono runtime.

You might want to consider this.